src/pp-annotator/PrzypisAdder.ts

```diff
@@ -13,35 +16,49 @@ const { $ } = util;
  * PrzypisAdder is for the most part a copy of annotator.Adder, except with two buttons, thus:
  * onCreate callback replaced with beginAnnotationCreate and beforeRequestCreate
  */
+
+interface IPrzypisAdderOptions extends annotator.ui.widget.IWidgetOptions {
+  beginAnnotationCreate?: (annotation: IAnnotation, e: JQuery.Event) => void;
+  beforeRequestCreate?: (annotation: IAnnotation, e: JQuery.Event) => void;
+}
+
 export default class PrzypisAdder extends Widget {
   NS = 'przypis-adder';
 
-  constructor(options) {
+  ignoreMouseup: boolean;
+  annotation: IAnnotation | null;
+  beginAnnotationCreate: IPrzypisAdderOptions["beginAnnotationCreate"];
+  beforeRequestCreate: IPrzypisAdderOptions["beforeRequestCreate"];
+  document: Document;
+
+  static options: IPrzypisAdderOptions;
```

Comment:
jeśli TS obsługuje class properties z es-next(a z tego co widzę w docsach, to tak), to tutaj można przerzucić linijki 189 - 193

Reply:
widzę niżej że w `PrzypisEditor` dokładnie tak zrobiłeś :)